ipv6/ipv6.go

```diff
@@ -47,19 +47,34 @@ var _ = IPv6Describe("IPv6 Connectivity Tests", func() {
 		},
 	}
 
-	describeIPv6Tests := func(buildpackPath string, stack string) {
+	describeIPv6Tests := func(buildpackPath, stack, manifestFile string) {
```

Comment:
"buildpackPath" is a bit confusing here. Shouldn't it be just "asset" or "assetPath"?